src/StochasticStyles/spawning.jl

```diff
@@ -171,14 +171,14 @@ struct Exact{T} <: SpawningStrategy
     Exact(threshold::T=0.0) where {T} = new{T}(threshold)
 end
 
-@inline function spawn!(s::Exact, w, offdiags::AbstractVector, add, val, boost=1)
+@inline function spawn!(s::Exact, w, column, add, val, boost=1)
     T = valtype(w)
-    spawns = sum(offdiags; init=zero(T)) do (new_add, mat_elem)
+    spawns = sum(column; init=zero(T)) do (new_add, mat_elem)
         abs(projected_deposit!(
             w, new_add, val * mat_elem, add => val, s.threshold
         ))
     end
-    return (length(offdiags), spawns)
+    return (length(column), spawns)#length might not be known
```

Comment:
```suggestion
    return (num_offdiagonals(column), spawns)
```
So this would either give the accurate answer, if known, or an estimate. A more ambitious way to do this would be to count the number of spawns while iterating through the column. This does come at the cost of a few CPU cycles, though, and maybe we don't need it.